### DISCUSSION TITLE
Time Series panel: add mouse-over action for legend values

### DISCUSSION THREAD

**Discussion by martingruening** [grafana/grafana#38235] [Dashboards, Panels & Visualizations]
I would love to see the following feature addition in the time series panel:
When I move the mouse over one of the legend fields (lets say 'Max') I would like to see the the point highlighted in the actual graph itself as if I moved the mouse over the point in the graph area. Example (red lines are only for illustration purposes added):

![Unbenannt](https://user-images.githubusercontent.com/36659742/109783609-789cef00-7c0a-11eb-9c06-d65fff118e67.PNG)
 
In this example, when I move the mouse over the max value in the right bottom corner, the highlighting should occur. If there are multiple graph values matching the legend value, they should also be highlighted (but without displaying the gray text boxes with time stamps and values). If no single graph values matches the legend values (i.e. for 'Mean') only a horizontal dotted line should be shown.

This would enable to speed up digging into interesting aspects of the graph much quicker than visual inspecting the graph to find the point in time when a special condition like 'Max' was reached. Today I hover about the graph with the mouse cursor a lot, to understand when exactly that specific datapoint was.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
